cristin-person/src/main/java/no/unit/nva/cristin/person/employment/delete/DeletePersonEmploymentClient.java

```diff
@@ -0,0 +1,86 @@
+package no.unit.nva.cristin.person.employment.delete;
+
+import no.unit.nva.cristin.model.SearchResponse;
+import no.unit.nva.cristin.person.employment.query.QueryPersonEmploymentClient;
+import no.unit.nva.cristin.person.model.cristin.CristinPersonEmployment;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+
+import static no.unit.nva.cristin.model.Constants.CRISTIN_API_URL;
+import static no.unit.nva.cristin.model.Constants.PERSON_PATH;
+import static nva.commons.core.attempt.Try.attempt;
+
+public class DeletePersonEmploymentClient extends QueryPersonEmploymentClient {
```

Comment:
A delete class should not extend a query class. I dont think we need to do a query before delete.